### PR TITLE
fix: make sure document is defined

### PR DESCRIPTION
**Pull request by KnisterPeter**
Check if global `document` exists to attach styles to.

Relates to #4

### CODE REVIEW

File: src/styles.ts [hunk 1]
```diff
@@ -32,18 +32,20 @@ export function addClassesForAllSizes(
...
-	const element = document.createElement('style');
-	element.type = 'text/css';
-	element.setAttribute('data-meta', prefix);
-	element.textContent = notAttachedCss;
-
-	// Add to the DOM
-	const container = document.querySelector('head');
-	if (!container) {
-		throw new Error('<head> tag not found');
+	if (typeof document !== 'undefined') {
```
Comment (sapegin) [inline src/styles.ts]: Would be nice to use an early return here instead of wrapping the whole thing in a condition.

**Review by sapegin — CHANGES_REQUESTED** [review]
Thanks!

**Comment by KnisterPeter** [comment]
@sapegin Updated as you recommended.

**Review by sapegin — APPROVED** [review]

**Comment by sapegin** [comment]
Awesome, merged!

Does it fix #4 or something else is needed?

**Comment by KnisterPeter** [comment]
I've not tested if it fully supports SSR

**Comment by KnisterPeter** [comment]
Thanks for merging

**Pull request merged**